--- hnb-1.8.1.orig/configure.in	Thu Nov  1 10:00:47 2001
+++ hnb-1.8.1/configure.in	Mon Nov  5 22:32:04 2001
@@ -7,13 +7,16 @@
 
 dnl Checks for libraries.
 dnl Replace `main' with a function in -lcurses:
-AC_CHECK_LIB(curses, move)
+if test -d /usr/include/ncurses ; then 
+   CFLAGS="$CFLAGS -I/usr/include/ncurses"
+fi
+AC_HAVE_LIBRARY(ncurses)
 AC_HAVE_LIBRARY(xml2)
 
-
 dnl Checks for header files.
 AC_HEADER_STDC
-AC_CHECK_HEADERS(fcntl.h unistd.h libxml/tree.h libxml/parser.h)
+AC_CHECK_HEADERS(fcntl.h unistd.h)
+AM_PATH_XML2
 
 dnl Checks for typedefs, structures, and compiler characteristics.
 AC_C_CONST
